src/BugsnagUnity.mm

```diff
@@ -109,6 +109,8 @@ void bugsnag_setNotifyUrl(const void *configuration, char *notifyURL) {
     return;
   NSString *ns_notifyURL = [NSString stringWithUTF8String: notifyURL];
   ((__bridge BugsnagConfiguration *)configuration).endpoints.notify = ns_notifyURL;
+  // Workaround for endpoint stale-cache issue: Force-trigger re-processing by reassinging endpoint
```

Comment:
Has a bug been raised for this on the Cocoa side? It'd be good to link to it if so, or create one if not.